internal/cmd/workspaces.go

```diff
@@ -752,3 +754,71 @@ func buildUpdateReq(ctx context.Context, client coder.Client, conf updateConf) (
 	}
 	return &updateReq, nil
 }
+
+func openWorkspaceCmd() *cobra.Command {
+	var (
+		workspaceName string
+	)
+
+	cmd := &cobra.Command{
+		Use:     "open",
+		Short:   "Open Coder workspace IDE",
+		Long:    "Open Coder workspace IDE in your default browser",
+		Example: "coder workspace open --name dev",
+		RunE: func(cmd *cobra.Command, args []string) error {
```

Comment:
We can add the ability to open the workspace with VS Code Remote locally.

1. Run `config-ssh` for them.
2. Ensure VS Code is installed.
3. Launch `vscode://vscode-remote/ssh-remote+coder.rem` scheme. (replace coder.rem with the target instance).